docs/general/ad-filtering/create-own-filters.md

```diff
@@ -233,6 +233,10 @@ This modifier lets you narrow the rule coverage down to a specific application (
 
 > For Mac, you can find out the bundle ID or the process name of the app by viewing the respective request details in the Filtering log.
 
+> **Compatibility with different versions of AdGuard**
+>
+> Starting with CoreLibs v1.12, the `$app` modifier for Windows is case-insensitive.
```

Comment:
This does not belong to the compatibility section. Just note that the names are case-insensitive (you can add a note that this behavior is from v1.12).

Reply:
actually we do write such notes like this